openml/extensions/sklearn/extension.py

```diff
@@ -883,7 +879,12 @@ def flatten_all(list_):
                             raise ValueError(msg)
                         else:
                             pass
-                    elif not isinstance(sub_component, (OpenMLFlow, type(None))):
+                    elif isinstance(sub_component, type(None)):
+                        msg = 'Cannot serialize objects of None type. Please replace with a '\
```

Comment:
Could you please add a unit test that serializing `None` indeed fails? Also, could you please phrase it as `Please use a valid placeholder for None`?

Reply:
Done.